frame/contracts/src/lib.rs

```diff
@@ -305,6 +313,11 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxStorageKeyLen: Get<u32>;
 
+		/// The maximum number of delegate_dependencies that a contract can lock with
+		/// [`chain_extension::Ext::add_delegate_dependency`].
+		#[pallet::constant]
+		type MaxDelegateDependencies: Get<u32>;
```

Comment:
wouldn't `Get<u8>` be enough here?

Reply:
who knows 255 might be too small, this is probably optimized by the compiler when you set a `ConstU32<N>`